source/operations/concepts/scanner.rst

```diff
@@ -0,0 +1,95 @@
+.. _minio-concepts-scanner:
+
+==============
+Object Scanner
+==============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+MinIO uses the built-in scanner to check objects for healing and to take any scheduled object actions.
+Such actions may include:
+
+- calculate data usage on drives
+- apply configured :ref:`lifecycle management <minio-lifecycle-management>` or :ref:`object retention <minio-object-retention>` rules
+- :ref:`bucket <minio-bucket-replication>` or :ref:`site <minio-site-replication-overview>` replication
+- check objects for missing or corrupted data or parity shards and perform :ref:`healing <minio-concepts-healing>`
+
+The scanner performs these functions at two levels: cluster and bucket.
+At the cluster level, the scanner splits all buckets into sets and scans one set at a time.
```

Comment:
Do we need to define set here?

Reply:
🤔 
I may just remove `sets` here to avoid conflation with `erasure sets`. Since these are different things.